lib/waterline/query/dql/update.js

```diff
@@ -114,21 +114,7 @@ function createBelongsTo(valuesObject, cb) {
     if(!modelName) return next();
 
     var model = self.waterline.collections[modelName];
-    var pkValue = valuesObject.originalValues[item][model.primaryKey];
-
-    var criteria = {};
-
-    var pkField = hop(model._transformer._transformations, model.primaryKey) ? model._transformer._transformations[model.primaryKey] : model.primaryKey;
-
-    criteria[pkField] = pkValue;
-
-    // If a pkValue if found, do a findOrCreate and look for a record matching the pk.
-    var query;
-    if(pkValue) {
-      query = model.findOrCreate(criteria, valuesObject.values[item]);
-    } else {
-      query = model.create(valuesObject.values[item]);
-    }
+    var query = model.create(valuesObject.values[item]);
```

Comment:
Just to clarify the change here - was Waterline secretly doing findOrCreates when you made certain other types of queries? Missing a bit of context around this block and the one in `create.js`